csharp/protos/extensions_issue6936_a.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
```

Comment:
I prefer a name that starts with `unittest_` to make it clear that this proto is used for unit tests. 
`unittest_issue6936_a.proto` or `unittest_issue6936_extensions_a.proto` both work.